CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 ## Unreleased
 
+## 2.0.0-pre1
+
+**NOTE:** This is a precursor release of triton v2.0.0, including backwards
+incompatible changes, like the modification of Instance's Metadata and Tags
+fields from `map[string][string]` and optionally `map[string]interface{}` to
+`map[string]interface{}` everywhere. This change suits better with the possible
+types of Metadata and Tags as defined by Triton's VMAPI: valid strings,
+booleans or numeric JSON values.
+
+- Fix List Instances type error due to Instance Metadata including boolean
+or numeric values [#166].
+
```

Comment:
Maybe make a note of code will need to reference using the `triton-go/v2` namespace?